test/support/action/resizeScreenSize.js

```diff
@@ -1,9 +1,16 @@
-module.exports = function (screenWidth, screenHeight, done) {
-    this.browser
-        .windowHandleSize({
-            width: parseInt(screenWidth),
-            height: parseInt(screenHeight)
-        })
-        .refresh()
-        .call(done);
+/**
+ * Resize the browser window
+ * @param  {String}   screenWidth  The width of the window to resize to
+ * @param  {String}   screenHeight The height of the window to resize to
+ * @param  {Function} done         Function to execute when finished
+ */
+module.exports = (screenWidth, screenHeight, done) => {
+    browser.windowHandleSize({
+        width: parseInt(screenWidth, 10),
+        height: parseInt(screenHeight, 10),
+    });
+
+    browser.refresh();
```

Comment:
I don't know why I had this in here but it seems obsolete for me to refresh the browser after resizing the window